mne/io/egi/egimff.py

```diff
@@ -555,7 +555,7 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                         data_view = data[n_data1_channels:, -1] = 0
                         warn('This file has the EGI PSG sample bug')
                         if self.annotations is None:
-                            self.annotations = Annotations((), (), ())
+                            self.set_annotations(Annotations((), (), ()))
```

Comment:
in base.py we make it None so I think we should make it None if there no annotations rather than empty like here.